Svc/FrameAccumulator/FrameAccumulator.cpp

```diff
@@ -118,17 +118,16 @@ void FrameAccumulator ::processRing() {
                   static_cast<FwAssertArgType>(m_inRing.get_allocated_size()), static_cast<FwAssertArgType>(remaining));
         // On successful detection, consume data from the ring buffer and place it into an allocated frame
         if (status == FrameDetector::FRAME_DETECTED) {
-            // size_out must be set to the size of the buffer and must fit within the existing data
+            // size_out must be set (non-zero) and must fit within the remaining data. Otherwise would mean
```

Comment:
Theoretically, a well-formed (but to large) packet could be sent from the ground.  Thus I would not assert here either.